test/k8sT/DatapathConfiguration.go

```diff
@@ -75,10 +74,6 @@ var _ = Describe("K8sDatapathConfig", func() {
 		}
 
 		blacklist := helpers.GetBadLogMessages()
-		if strings.Contains(CurrentGinkgoTestDescription().TestText, "sockops") {
-			delete(blacklist, helpers.ClangErrorsMsg)
-			delete(blacklist, helpers.ClangErrorMsg)
-		}
 		kubectl.ValidateListOfErrorsInLogs(CurrentGinkgoTestDescription().Duration, blacklist)
```

Comment:
Should `blacklist` and `kubectl.ValidateListOfErrorsInLogs()` be kept here, or should commit d7e4299bcf47 be reverted? /cc @pchaigno. Since it's no longer used, it would need a comment at the very least.

Reply:
Commit d7e4299 shouldn't be reverted, but `ValidateListOfErrorsInLogs` can be replaced by `ValidateNoErrorsInLogs`.